src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -18,7 +18,7 @@
      Generates Program file which contains the Main entry point
      ============================================================
   -->
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Comment:
No, It will create only first time. Subsequent build and rebuild will use the same